ironfish-rust/src/keys/public_address.rs

```diff
@@ -179,7 +150,8 @@ impl PublicAddress {
     /// another because `pk_d` is not a name I want to expose in a public
     /// interface.
     pub(crate) fn sapling_payment_address(&self) -> PaymentAddress {
```

Comment:
do we still need this sapling payment address? should we call public_address when we need a payment address?